clients/src/main/java/org/apache/kafka/common/security/kerberos/Login.java

```diff
@@ -286,11 +286,11 @@ public Subject subject() {
     private synchronized LoginContext login(final String loginContextName) throws LoginException {
         String jaasConfigFile = System.getProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM);
         if (jaasConfigFile == null) {
-            throw new IllegalArgumentException("You must pass " + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + " in secure mode.");
+            log.info("System property '" + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' is not set, using default JAAS configuration.");
```

Comment:
Also, I think in the exception thrown if `configEntries == null` should mention whether the system property is set or not as this would be helpful for people who thought it was set.